src/App/Entity/Meetup.php

```diff
@@ -63,7 +63,12 @@
     private $talks;
 
     /**
-     * @ORM\OneToMany(targetEntity=MeetupAttendee::class, mappedBy="meetup")
+     * @ORM\OneToMany(
+     *     targetEntity=MeetupAttendee::class,
+     *     mappedBy="meetup",
+     *     orphanRemoval=true,
+     *     cascade={"persist", "remove"}
```

Comment:
I haven't used orphan removal, so I'm not familiar with it. Does it make sense to use it with cascade remove?